examples/image-streams/image-streams-centos7.json

```diff
@@ -904,40 +904,20 @@
           },
           {
             "annotations": {
-              "description": "Build and run Ruby 2.5 applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-ruby-container/blob/master/2.5/README.md.",
+              "description": "Build and run Ruby 2.7 applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-ruby-container/blob/master/2.6/README.md.",
               "iconClass": "icon-ruby",
-              "openshift.io/display-name": "Ruby 2.5",
+              "openshift.io/display-name": "Ruby 2.7",
               "openshift.io/provider-display-name": "Red Hat, Inc.",
               "sampleRepo": "https://github.com/sclorg/ruby-ex.git",
-              "supports": "ruby:2.5,ruby",
+              "supports": "ruby:2.7,ruby",
               "tags": "builder,ruby",
-              "version": "2.5"
-            },
-            "from": {
-              "kind": "DockerImage",
-              "name": "registry.centos.org/centos/ruby-25-centos7:latest"
-            },
-            "name": "2.5",
-            "referencePolicy": {
-              "type": "Local"
-            }
-          },
-          {
-            "annotations": {
-              "description": "Build and run Ruby 2.6 applications on CentOS 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-ruby-container/blob/master/2.6/README.md.",
-              "iconClass": "icon-ruby",
-              "openshift.io/display-name": "Ruby 2.6",
-              "openshift.io/provider-display-name": "Red Hat, Inc.",
-              "sampleRepo": "https://github.com/sclorg/ruby-ex.git",
-              "supports": "ruby:2.6,ruby",
-              "tags": "builder,ruby",
-              "version": "2.6"
+              "version": "2.7"
             },
             "from": {
               "kind": "DockerImage",
-              "name": "registry.centos.org/centos/ruby-26-centos7:latest"
+              "name": "registry.centos.org/centos/ruby-27-centos7:latest"
```

Comment:
The image does not exist, could we push it firstly?
$skopeo inspect docker://registry.centos.org/centos/ruby-27-centos7:latest
FATA[0002] manifest unknown: manifest unknown

Reply:
If this is the case, then I don't think we are using this example in test code. If that is the case, I would remove it as this capability has been replaced by the samples operator.